src/common/exif.cc

```diff
@@ -572,6 +572,12 @@ static bool dt_exif_read_exif_data(dt_image_t *img, Exiv2::ExifData &exifData)
     {
       dt_strlcpy_to_utf8(img->exif_lens, sizeof(img->exif_lens), pos, exifData);
     }
+    else if(Exiv2::testVersion(0,25,0)
```

Comment:
Did you check that this will only be available in the 0.25? If 0.24 already has it we can use that instead.